lib/lte_link_control/Kconfig

```diff
@@ -19,6 +19,11 @@ config LTE_AUTO_INIT_AND_CONNECT
 		automatically initialize and connect the modem
 		before the application starts
 
+config LTE_SHELL
+	bool "Enable lte shell commands"
+	default y
```

Comment:
Because this is enabled by default when `CONFIG_SHELL` is enabled, I think it would be good to modify modem_shell prj.conf in this PR and disable `CONFIG_LTE_SHELL`. modem_shell has own more comprehensive shell implementation for LTE link control, so this is not needed.

Reply:
Fixed.